<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Example One</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="new_user_group_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>New User</string>
    </property>
    <widget class="QLabel" name="new_user_name_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_user_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="new_password_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_user_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="login_group_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Log In</string>
    </property>
    <widget class="QLabel" name="login_user_name_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_user_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="login_password_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
